### DISCUSSION TITLE
Response DTO 분리

### DISCUSSION THREAD

**Discussion by softpeanut** [Team-ComIT/SimTong-Backend#91] [의논하기]
## AS-IS
- application 또는 domain module에서 Response DTO를 생성해 presentation module로 반환한다.
- presentation module에서는 해당 Response DTO를 요청자에게 그대로 반환한다.

## Discussion
- Response DTO를 생성하는 위치를 presentation module로 바꿀지 고민

**Comment by khcho0125** [comment]
하나의 데이터를 Return 할 때는 문제가 없겠지만 여러 데이터를 Return 받을 수단이 있어야 할 듯합니다.
